nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -4,6 +4,10 @@ with lib;
 
 let
   cfg = config.services.xserver.windowManager.i3;
+  updateSessionEnvironmentScript = ''
+    systemctl --user import-environment PATH DISPLAY XAUTHORITY DESKTOP_SESSION XDG_CONFIG_DIRS XDG_DATA_DIRS XDG_RUNTIME_DIR XDG_SESSION_ID XDG_CONFIG_DIRS || true
```

Comment:
`XDG_CONFIG_DIRS` is duplicated

According to `man dbus-update-activation-environment`, should we add `DBUS_SESSION_BUS_ADDRESS` to the systemd command?

```
Variables that are special to dbus-daemon or systemd may be set, but their values will be overridden when a service is started. For instance, it is not useful
to add DBUS_SESSION_BUS_ADDRESS to dbus-daemon's activation environment, although it might still be useful to add it to systemd's activation environment.
```

Thanks

Reply:
Fixed the duplicate and added DBUS_SESSION_BUS_ADDRESS